### PR TITLE
Apps: add settings.dev.mozaws.net(kinto-dev) app

### PR DESCRIPTION
**Pull request by rvaralda**
add settings.dev.mozaws.net(kinto-dev) app

### CODE REVIEW

File: apps.yml [hunk 1]
```diff
@@ -4144,3 +4144,15 @@ apps:
...
+    - hris_is_staff
+    - team_moco
+    - team_mofo
+    authorized_users: []
+    client_id: 7M49iYODCiCGdldu4awmF2Pr6pHsRZDe
+    display: false
+    logo: auth0.png
+    name: settings.dev.mozaws.net
+    op: auth0
+    url: https://settings.dev.mozaws.net/v1/admin/
```
Comment (floatingatoll) [inline apps.yml:4158]: Should this be the auth0/samlp URL with the client_id in it?

Comment (rvaralda) [reply apps.yml:4158]: In this case I just keep the same config as other environments.
i.e. https://github.com/mozilla-iam/sso-dashboard-configuration/blob/master/apps.yml#L2525

**Comment by floatingatoll** [comment]
Ah, since there’s no redirect shorturl, and no tile displayed, the URL
probably isn’t used at all. It should be fine then.

**Review by floatingatoll — APPROVED** [review]

**Comment by bkochendorfer** [comment]
@rvaralda commit looks good but you need to sign it https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits

**Pull request merged**